### ISSUE TITLE
Uptime not correct

### ISSUE THREAD

**Issue by ingostrohmeier** [MikeJewski/ArduinoDeck#8]
Uptime Count in the top bar on the deck isn't starting at 0:00. Most of the time it starts at 5:00, sometimes at 2:00 (example: it shows 05:12 after 12 minutes of streaming)

**Comment by MikeJewski** [comment]
Is this still happening? If it is, is it happening every time?

**Comment by ingostrohmeier** [comment]
Yes, still happening every time I start streaming. I've reuploaded the code to the Arduino, but its still the same.

**Comment by MarcTowler** [comment]
Definitely still every time
